i18n/en_US.csv

```diff
@@ -0,0 +1 @@
+"Your order # is: <span>%1</span>.","Your Order Number is: <span>%1</span>.",module,Magento_Checkout
```

Comment:
never add html tags to translations.
find a way, how to avoid this.

this working only for modules, themes' .csv files don't support module directives for translations
```
,module,Magento_Checkout
```

Reply:
Ok I asked Matt about this and he told me it was not good either, but that it was necessary this time. He told me that if a remove the html tags the translation will break.